business_requirement_deliverable_cost/report/br_deliverable_cost_report.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import tools
+from openerp import fields, models
+
+
+class BusinessRequirementDeliverableCostReport(models.Model):
+    _name = "business.requirement.deliverable.cost.report"
+    _description = "Deliverable Cost Report"
+    _auto = False
+
+    name = fields.Char('Name', readonly=True)
+    description = fields.Char('Description', readonly=True)
+    partner_id = fields.Many2one('res.partner', 'Customer', readonly=True)
+    project_id = fields.Many2one('project.project', 'Master Project',
+                                 readonly=True)
+    change_request = fields.Boolean('Change Request?', readonly=True)
+    priority = fields.Selection([('0', 'Low'), ('1', 'Normal'),
+                                 ('2', 'High')], 'Priority', readonly=True)
+    dlv_description = fields.Text('Deliverable Description', readonly=True)
+    dlv_product = fields.Many2one('product.product', 'Dlv Product',
+                                  readonly=True)
+    res_description = fields.Text('Resource Description', readonly=True)
+    res_product = fields.Many2one('product.product', 'Res Product',
+                                  readonly=True)
+    br_count = fields.Integer('BR Count', readonly=True)
+    dlv_count = fields.Integer('Deliverable Count', readonly=True)
+    res_count = fields.Integer('Resource Count', readonly=True)
+    dlv_qty = fields.Float('Deliverable Qty', readonly=True)
+    res_qty = fields.Float('Resource Qty', readonly=True)
+    sale_price = fields.Float('Sale Price', readonly=True)
+    total_revenue = fields.Float('Total Revenue', readonly=True)
+    cost_price = fields.Float('Cost price', readonly=True)
+    total_cost = fields.Float('Total cost', readonly=True)
```

Comment:
:+1: 
Functional tests are fine but I have 2 small additional requests:
* Can you add Gross Profit field? It is Total revenue - Total cost
* Can add the description of the BR (after name)